src/Pages/Festival/Festival.ts

```diff
@@ -0,0 +1,6 @@
+export interface Festival {
```

Comment:
Rename this file to `types.ts`. It is not a react component and so file name should not start with a capital letter